python/felis/cli.py

```diff
@@ -337,19 +307,16 @@ def load_tap_schema(
     The TAP_SCHEMA database must already exist or the command will fail. This
     command will not initialize the TAP_SCHEMA tables.
     """
-    url = make_url(engine_url)
-    engine: Engine | MockConnection
-    if dry_run or is_mock_url(url):
-        engine = create_mock_engine(url, executor=None)
-    else:
-        engine = create_engine(engine_url)
+    # Create TableManager with automatic dialect detection
     mgr = TableManager(
-        engine=engine,
-        apply_schema_to_metadata=False if engine.dialect.name == "sqlite" else True,
+        engine_url=engine_url,
         schema_name=tap_schema_name,
         table_name_postfix=tap_tables_postfix,
     )
 
+    # Create DatabaseContext using TableManager's metadata
+    db_ctx = create_database_context(engine_url, mgr.metadata, dry_run=dry_run)
```

Comment:
Should this be passing in echo & output_file as well or is this by design?

Reply:
Good catch - it should be passing these arguments. I'll add them.